app.js

```diff
@@ -1,9 +1,26 @@
 import express from "express";
+import cors from "cors";
+
+import { AuthorizationController } from "./controllers/authorization.controller.js";
+
+import { checkToken } from "./middleware/token.middleware.js";
+import { WeatherController } from "./controllers/weather.controller.js";
+
+const { CORS_ORIGIN } = process.env;
 
 const app = express();
 
-app.get("/", (req, res) => {
-  res.send("works");
-});
+app.use(
```

Comment:
Please add helmet npm library. you can use default config this library.
https://helmetjs.github.io/
Also add `app.disable('x-powered-by');` to hide header showing that we use express